### PR TITLE
Updated Pixie Requirements

### PR DESCRIPTION
**Pull request by lchapman4**
Added a blurb about Pixie needing to run privileged. Also added a note on installing our K8s integration with privileged=false which prevents Pixie from running.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/kubernetes-pixie/auto-telemetry-pixie/install-auto-telemetry-pixie.mdx [hunk 1]
```diff
@@ -28,6 +28,7 @@ In addition:
 
 * Review this [Pixie data security overview](/docs/auto-telemetry-pixie/pixie-data-security-overview) for actions to take to secure your data.
 * Make sure you have sufficient memory: Pixie requires, at least, 2Gb of memory on each node in your cluster. More memory may be required for larger clusters.
+* Pixie requires elevated privileges. By default, installing the NR Kubernetes integration with privileged=falese will also prevent Pixie from running. See [Step 3 here](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#installing-and-configuring-nri-bundle-with-helm)
```
Comment (shawnj) [inline src/content/docs/kubernetes-pixie/auto-telemetry-pixie/install-auto-telemetry-pixie.mdx:31]: Looks good.  Please update spelling `privileged=falese` to `privileged=false`. :)

Comment (barbnewrelic) [reply src/content/docs/kubernetes-pixie/auto-telemetry-pixie/install-auto-telemetry-pixie.mdx]: I'll fix with some minor styleguide edits

**Comment by lchapman4** [comment]
Hold off on merging this please. I need to update this PR. Thanks.

**Comment by lchapman4** [comment]
Ok, this PR is ready now. Merge whenever is convenient.

**Review by barbnewrelic — APPROVED** [review]
Thanks for adding this helpful requirements info!

**Pull request merged**